factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/registry/transformation/TransformationStore.java

```diff
@@ -21,6 +21,10 @@
     void register(TransformationSource source, String transformation)
             throws TransformationConflictException;
 
+    void register(TransformationRegistrationListener listener);
```

Comment:
A little bit confusing that `register` is used for both, sources and listeners 🤔

Reply:
agreed. will rename RegisterListener to StoreListener and rename methods accordingly